mlir/lib/Dialect/MIOpen/Generator/IsaNameParser.cpp

```diff
@@ -0,0 +1,128 @@
+//===- IsaNameParser.cpp - MLIR to C++ option parsing ---------------===//
+//
+// Part of the MLIR Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file implements isa name string parser
+//
+//===----------------------------------------------------------------------===//
+
+#include "mlir/Dialect/MIOpen/Generator/IsaNameParser.h"
+#include "llvm/Support/Error.h"
+
+#include <cstring>
+#include <numeric>
```

Comment:
could you share insights why `<numeric>` is needed here?

Reply:
It is there because I used `std::accumulate` to assemble up the feature string below:

https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/blob/f53f5226693392fedcee058aa7b7ff9d0ebdfc2c/mlir/lib/Dialect/MIOpen/Generator/IsaNameParser.cpp#L85-L91